Aztec/Classes/TextKit/TextView.swift

```diff
@@ -909,23 +909,42 @@ open class TextView: UITextView {
 
         let originalText = attributedText.attributedSubstring(from: range)
         let attributedTitle = NSAttributedString(string: title)
-        let finalRange = NSRange(location: range.location, length: attributedTitle.length)        
+        let finalRange = NSRange(location: range.location, length: attributedTitle.length)
 
         undoManager?.registerUndo(withTarget: self, handler: { [weak self] target in
             self?.undoTextReplacement(of: originalText, finalRange: finalRange)
         })
 
         let formatter = LinkFormatter()
-        formatter.attributeValue = url        
+        formatter.attributeValue = url
         let attributes = formatter.apply(to: typingAttributes)
+        let underLineStr = addUnderlineInLink(title: title, attributes: attributes)
```

Comment:
As a side note, the `LinkFormatter` class should also have additional code to be able to remove the underline from links too.